Assets/Scripts/Tank/TankView.cs

```diff
@@ -30,7 +32,7 @@ public void TakeDamage(float Damage)
         void Start()
         {
             transform.position = transform.position.SetY(-3.74f);
-            EventService.Instance.EnemyOnDeath += AddTankHealth;
+            EventService.Instance.EnemyOnDeath += AddTankHealth; // Sub to event when enemy dies it gets health

```

Comment:
the comment is pretty useless here 
// // Sub to event when enemy dies it gets health

since the code itself is very easy to read and variables are named correctly

Reply:
remove comments